### PR TITLE
Registry system node updates fixes #475 

### LINKED ISSUE #475: Update Nodes service
Update the node service in order to support new set of requirements (sites).

### PR DESCRIPTION
**Pull request by delvatt**
This pull request fixes #475

### CODE REVIEW

File: systems/registry/node/pkg/server/event.go [hunk 1]
```diff
@@ -26,16 +26,16 @@ func NewNodeEventServer(nodeRepo db.NodeRepo) *NodeEventServer {
...
-		msg, err := n.unmarshalNodeOnlineEvent(e.Msg)
-		if err != nil {
-			return nil, err
-		}
-
-		err = n.handleNodeOnlineEvent(e.RoutingKey, msg)
-		if err != nil {
-			return nil, err
-		}
+	// case "event.cloud.node.node.online":
```
Comment (vthakur7f) [inline systems/registry/node/pkg/server/event.go:29]: Any reason why this is commented ?

File: systems/registry/node/pb/node.proto [hunk 2]
```diff
@@ -20,108 +26,119 @@ service NodeService {
 }
 
 
-message AddNodeToNetworkRequest {
-    string node = 1;
-    string network = 2;
+message AddNodeRequest{
+    string nodeId = 1 [(validator.field) = {string_not_empty: true}];
+    string orgId = 2 [(validator.field) = {uuid_ver: 4, string_not_empty: true}, json_name = "org_id"];
+	string name = 3;
```
[the remark this answers is not in the document]
Comment (delvatt) [reply systems/registry/node/pb/node.proto]: fixed

File: systems/registry/node/pb/node.proto [hunk 3]
```diff
@@ -20,108 +26,119 @@ service NodeService {
...
+message ReleaseNodeFromNetworkRequest {
+    string nodeId = 1 [(validator.field) = {string_not_empty: true}, json_name = "node_id"];
 }
 
+message ReleaseNodeFromNetworkResponse {
+
+}
 
+message Node {
+    string id = 1 [(validator.field) = {string_not_empty: true}];
```
[the remark this answers is not in the document]
Comment (delvatt) [reply systems/registry/node/pb/node.proto:141]: Thanks for the notice.

File: systems/registry/node/pb/node.proto [hunk 4]
```diff
@@ -20,108 +26,119 @@ service NodeService {
...
 
-message ReleaseNodeFromNetworkRequest {
-    string node = 1;
+message GetNodeResponse{
+    Node node = 1;
 }
 
-message ReleaseNodeFromNetworkResponse {
 
+message GetBySiteRequest{
```
[the remark this answers is not in the document]
Comment (delvatt) [reply systems/registry/node/pb/node.proto:51]: fixed

File: systems/registry/node/pkg/db/node_repo.go [hunk 5]
```diff
@@ -62,140 +68,103 @@ func (r *nodeRepo) Get(id ukama.NodeID) (*Node, error) {
...
+	return nodes, nil
 }
 
-func (r *nodeRepo) Delete(id ukama.NodeID, nestedFunc ...func() error) error {
-	err := r.Db.ExecuteInTransaction(func(tx *gorm.DB) *gorm.DB {
-		d := tx.Delete(&Node{}, "node_id = ?", id.StringLowercase())
-
-		if d.Error != nil {
-			return d
+// TODO: check for still allocated and attached nodes
```
Comment (vthakur7f) [inline systems/registry/node/pkg/db/node_repo.go:95]: you can add issue for this.

**Review by vthakur7f — APPROVED** [review]
LGTM..!!

**Pull request merged**